packages/babel-helper-wrap-function/src/index.ts

```diff
@@ -89,37 +90,50 @@ function classOrObjectMethod(
 }
 
 function plainFunction(
-  path: NodePath<Exclude<t.Function, t.Method>>,
+  inPath: NodePath<Exclude<t.Function, t.Method>>,
   callId: t.Expression,
   noNewArrows: boolean,
   ignoreFunctionLength: boolean,
 ) {
-  let functionId = null;
+  let path: NodePath<
+    | t.FunctionDeclaration
+    | t.FunctionExpression
+    | t.CallExpression
+    | t.ArrowFunctionExpression
+  > = inPath;
   let node;
-  if (path.isArrowFunctionExpression()) {
+  let functionId = null;
+  const nodeParams = inPath.node.params;
+
+  if (inPath.isArrowFunctionExpression()) {
```

Comment:
Can we always use `path` instead of `inPath` (i.e. only use `inPath` for the original `let path = inPath` declaration)? I was initially confused about what's the difference between the two variables, before realizing that it's just because `path` can later become a `CallExpression` but we only want to allow functions as parameters.